### PR TITLE
 [FLINK-6720] Activate strict checkstyle in flink-java8

### PR DESCRIPTION
**Pull request by zentol**

### CODE REVIEW

File: flink-java8/src/test/java/org/apache/flink/test/api/java/operators/lambdas/FilterITCase.java [hunk 1]
```diff
@@ -35,27 +38,38 @@
...
+		data.add(new Tuple3<>(2, 2L, "Hello"));
+		data.add(new Tuple3<>(3, 2L, "Hello world"));
+		data.add(new Tuple3<>(4, 3L, "Hello world, how are you?"));
+		data.add(new Tuple3<>(5, 3L, "I am fine."));
+		data.add(new Tuple3<>(6, 3L, "Luke Skywalker"));
+		data.add(new Tuple3<>(7, 4L, "Comment#1"));
+		data.add(new Tuple3<>(8, 4L, "Comment#2"));
+		data.add(new Tuple3<>(9, 4L,
+			"Comment#3"));
+		data.add(new Tuple3<>(10, 4L
```
Comment (zentol) [inline flink-java8/src/test/java/org/apache/flink/test/api/java/operators/lambdas/FilterITCase.java]: fix line breaks.

File: flink-java8/src/test/java/org/apache/flink/test/api/java/operators/lambdas/FilterITCase.java [hunk 2]
```diff
@@ -35,27 +38,38 @@
...
-		data.add(new Tuple3<Integer, Long, String>(21,6l,"Comment#15"));
+		data.add(new Tuple3<>(1, 1L, "Hi"));
+		data.add(new Tuple3<>(2, 2L, "Hello"));
+		data.add(new Tuple3<>(3, 2L, "Hello world"));
+		data.add(new Tuple3<>(4, 3L, "Hello world, how are you?"));
+		data.add(new Tuple3<>(5, 3L, "I am fine."));
+		data.add(new Tuple3<>(6, 3L, "Luke Skywalker"));
+		data.add(new Tuple3<>(7, 4L, "Comment#1"));
+		data.add(new Tuple3<>(8, 4L, "Comment#2"));
+		data.add(new Tuple3<>(9, 4L,
```
Comment (greghogan) [inline flink-java8/src/test/java/org/apache/flink/test/api/java/operators/lambdas/FilterITCase.java]: Formatting.

**Comment by zentol** [comment]
@greghogan should be fixed now.

**Comment by greghogan** [comment]
+1

**Comment by zentol** [comment]
merging.

**Pull request closed without merging**